usecases/executor_factory/helpers.go

```diff
@@ -42,4 +42,11 @@ func TransactionReturnValue[ReturnType any](
 type TransactionFactory interface {
 	TransactionInOrgSchema(ctx context.Context, organizationId string, f func(tx repositories.Executor) error) error
 	Transaction(ctx context.Context, fn func(tx repositories.Executor) error) error
+	// Transaction(ctx context.Context, fn func(tx repositories.Executor) error) error
+}
+
+// Interface to be used in usecases, implemented by the DbExecutorFactory class in the usecases/db_executor_factory package
+// which itself has the ExecutorGetter repository class injected in it.
+type ClientSchemaExecutorFactory interface {
```

Comment:
I had to move this back here (from usecases) because of circular dependencies

Reply:
No problem, and I think we can create a `MarbleSchemaExecutorFactory` or something like that (linked to another comment on the `isNil`)